editor/import/resource_importer_obj.cpp

```diff
@@ -168,18 +168,23 @@ Error ResourceImporterOBJ::import(const String &p_source_file, const String &p_s
 
 					if (face[idx].size() == 3) {
 						int norm = face[idx][2].to_int() - 1;
+						if (norm < 0)
+							norm += normals.size() + 1;
```

Comment:
if the index is negative, it start form -1
and the line before is 
int norm = face[idx][2].to_int() - 1;
if norm is -1, it would be normals.size() - 1 - 1 + 1 = normals.size() - 1